crates/script/src/verify.rs

```diff
@@ -106,6 +106,10 @@ impl VerifyBundle {
         libraries: &[String],
     ) -> Option<VerifyArgs> {
         for (artifact, contract) in self.known_contracts.iter() {
+            // Avoid comparing to empty bytecode.
+            if contract.bytecode.is_empty() {
```

Comment:
we don't need this check if we add the check in `ContractsByArtifact` i think?

Reply:
yeah, but I've seen this bug before so would like to keep this as impact of this bug is pretty high and it's pretty easy to miss while working with artifacts

Reply:
also I think would've been caught by verification tests which are disable rn

cc @mattsse